universe/interface.go

```diff
@@ -552,6 +552,44 @@ type FederationLog interface {
 	LogNewSyncs(ctx context.Context, addrs ...ServerAddr) error
 }
 
+// ProofType is an enum that describes the type of proof which can be stored in
+// a given universe.
+type ProofType int
+
+const (
+	// ProofTypeNone indicates that all proofs should be excluded from the
+	// universe.
+	ProofTypeNone ProofType = iota
```

Comment:
I think maybe instead issuance should be the default? 

In what scenario would you imagine this is used, and doesn't return an error?

Reply:
I think this will be used for the setting of "what proofs do I want to sync with my universe", which can be "none". But also still going through the diff.

Reply:
> "what proofs do I want to sync with my universe", which can be "none".

Yes, that's what I had in mind. `ProofTypeNone` also allows us to not specify the proof type in an RPC call.